src/main/java/seedu/address/model/recommendation/Recommendation.java

```diff
@@ -0,0 +1,140 @@
+package seedu.address.model.recommendation;
+
+import seedu.address.model.location.Location;
+import seedu.address.model.person.ContactIndex;
+import seedu.address.model.time.TimePeriod;
+
+/**
+ * A meetup suggestion consisting of a location and time period.
+ */
+public class Recommendation implements Comparable<Recommendation> {
+    private final Location location;
+    private final TimePeriod timePeriod;
+    private final ContactIndex contactIndex;
```

Comment:
So for this contactindex is it for person or only for recommendation, the naming is confusing.